### PR TITLE
Grammatical changes

### PR DESCRIPTION
**Pull request by lysackm**
Some small phrase changes.

### CODE REVIEW

File: src/pages/Home/Home.tsx [hunk 1]
```diff
@@ -53,19 +53,21 @@ export const Home = () => {
                     concerns of CS students to the CS department, the Faculty of
                     Science, and the university administration. We work to
                     ensure that CS students have a high-quality education and a
-                    positive academic experience. We also strive to improve the
-                    CS field and society by promoting diversity, inclusion, and
+                    positive academic experience. We strive to improve the
+                    CS field by promoting diversity, inclusion, and
```
Comment (mbattad) [inline src/pages/Home/Home.tsx:57]: do we want to favour "computer science" over "CS" throughout? i do like replacing the acronym with the full term i wonder if it's worth being consistent with it. also like that you omitted "cs students" in favour of just "students" later on, i'd be cool with doing that too

+not a nitpick with your changes, but i hadn't noticed how similar the advocacy blurb is to the main one hahaha. not a huge deal but i can mess with it a little bit later on to restate less info

File: src/pages/Home/Home.tsx [hunk 2]
```diff
@@ -53,19 +53,21 @@ export const Home = () => {
...
             <Teaser title="Community" image="/img/teasers/community.jpg">
                 <Text>
                     We are a community of computer science students who
                     love technology and learning. We help CS students connect
-                    with each other and with the CS field. We celebrate the
+                    with each other and the industry. We celebrate the
                     diversity and achievements of our CS community. We also
                     organize events and activities for CS students to network,
-                    learn, and have fun. Come join us and visit our CS lounge!
+                    where they will hopefully build relationships that last
```
Comment (mbattad) [inline src/pages/Home/Home.tsx:68]:
```suggestion
                    where they can build relationships that last 
```
manifesting 😎

**Review by mbattad — COMMENTED** [review]
other than my tiny nitpicks LGTM thank you for fixing it up! and yeah of course if people don't think my points are important enough then go ahead and merge 👍  im not tryna request changes HAHAHA

File: src/pages/Home/Home.tsx (hunk 1 above)
Comment (lysackm) [reply src/pages/Home/Home.tsx:57]: Yes its very similar and I think changing it makes sense. I think changing CS to computer science is a good idea

**Pull request merged**